handlers/handlers.go

```diff
@@ -40,68 +42,98 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-func forwardFlorenceTokenIfRequired(req *http.Request) *http.Request {
-	if len(req.Header.Get(common.FlorenceHeaderKey)) > 0 {
-		ctx := common.SetFlorenceIdentity(req.Context(), req.Header.Get(common.FlorenceHeaderKey))
-		return req.WithContext(ctx)
-	}
-	return req
+func setStatusCode404(req *http.Request, w http.ResponseWriter, err error) {
+	status := http.StatusNotFound
+	log.ErrorCtx(req.Context(), err, log.Data{"setting-response-status": status})
+	w.WriteHeader(status)
 }
 
-//GeographyRender ...
-func GeographyRender(rend RenderClient) http.HandlerFunc {
+//GeographyHomepageRender ...
+func GeographyHomepageRender(rend RenderClient) http.HandlerFunc {
 
 	return func(w http.ResponseWriter, req *http.Request) {
 		var page geographyHomepage.Page
 
-		resp, err := http.Get(`https://api.dev.cmd.onsdigital.co.uk/v1/code-lists` + localAuthority)
+		homePageLink := url + `/code-lists` + localAuthority
+
+		resp, err := http.Get(homePageLink)
 		if err != nil {
+			log.Error(err, log.Data{"error getting data from the code-lists api http.Get(" + homePageLink + ") for GeographyHomepageRender returned ": err})
```

Comment:
Immediately after the `if err != nil {}` block you need to add a `defer resp.Body().Close()`. Really easy to forget to do - I do all the time - but it can cause some nasty non obvious bugs so best practice is to try to always add a defer close immediately after you open something.